public/js/core.js

```diff
@@ -59,8 +59,13 @@ tarchive.controller('CoreController', ['$scope', '$filter', 'API', 'NgTableParam
 
 	$scope.askForKey = function() {
 		swal({
-			title: "Enter your key!",
-			text: "Enter the key provided by the bot:",
+			html: true,
+			title:	'Welcome to Tarchivepanel!',
+			text:		'What is Tarchivebot? <br><br> A Telegram bot that archives your chat and provides you with a web interface to look up messages and statistics. <br><br> Add the bot using this link: ' +
+							'<a href="https://t.me/Tarchivebot" target="_blank">Tarchivebot</a><br>' +
```

Comment:
I'd be fine with this being in another PR unless @holzit want's to do it directly in this PR already.